blocks/search/search-api.js

```diff
@@ -1,8 +1,10 @@
 import { SEARCH_URLS } from '../../scripts/common.js';
 
 const { SEARCH_URL_DEV, SEARCH_URL_PROD } = SEARCH_URLS;
-const isProd = !window.location.host.includes('hlx.page') && !window.location.host.includes('localhost');
-const SEARCH_LINK = !isProd ? SEARCH_URL_DEV : SEARCH_URL_PROD;
+const isProd = !window.location.host.includes('hlx.page')
```

Comment:
as an improvement, WDYT about make this a 1Liner like the following:
`const isProd = ['hlx.page', 'aem.page', 'localhost'].every(substring => !window.location.host.includes(substring));` if not, it's ok like this

Reply:
I will keep it like this mostly because I need to merge this and not leave the focus of what I am doing now. But thanks :)